test/nopoll-regression-client.c

```diff
@@ -1678,8 +1678,8 @@ nopoll_bool test_17_send_and_receive_test (noPollCtx * ctx, noPollConn * conn, n
 
 	/* printf ("Test 17: sending partial content..\n"); */
 	_socket = nopoll_conn_socket (conn);
-	buffer[0] = 129;
-	buffer[1] = 150;
+	buffer[0] = 126;
+	buffer[1] = 127;
```

Comment:
I see that buffer is a `char`, so that's probably a compiler warning/error, but why change them from 129 -> 126 and 150 -> 127?  Do the values mean anything?

Reply:
hang on a moment. It seems that buffer should really be defined as unsigned char, since char is platform specific
restricting youself to 127 squeezes you just inside a max positive value with char but I think we need to take a look
at the type to begin with.

Reply:
My first instinct was to change `buffer`'s datatype as well. Since it was a unit test, I had figured that changing the values was less "intrusive". :-)

Reply:
From the history, https://github.com/Comcast/nopoll/commit/a1d34de80769fdb83d1338980917d24c6ef9fca4 there is no indication why those values in `buffer` had to be `> 127`, or why they cannot be `<= 127`.